sources/meta-yoe/dynamic-layers/meta-ti/recipes-kernel/linux/linux-ti-staging/updater.cfg

```diff
@@ -0,0 +1 @@
+CONFIG_USB=y
```

Comment:
Thanks for adding this!

Reply:
it seems I need to add few more options keep tuned.

Reply:
looks like that should help -- thanks!